src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2039,6 +2039,23 @@ SpinelNCPInstance::property_set_value(
 				);
 			}
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_IPv6MeshLocalPrefix)) {
+			struct in6_addr address = any_to_ipv6(value);
+
+			Data command =
+				SpinelPackData(
+					SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_DATA_S),
+					SPINEL_PROP_IPV6_ML_PREFIX,
+					address.s6_addr,
+					8
```

Comment:
The code here is encoding the property as a data blob of 8 bytes. But `spinel.h` and docs here [spinel-prop-ipv6.md](https://github.com/openthread/openthread/blob/master/doc/spinel-protocol-src/spinel-prop-ipv6.md#prop-98-prop_ipv6_ml_prefix-prop-ipv6-ml-prefix), define `SPINEL_PROP_IPV6_ML_PREFIX` with encoding of `[6C]` (an IPv6 address followed by a prefix length). Let's change this to match. Here is example of this from OpenThread' `NcpBase` [set handler](https://github.com/openthread/openthread/blob/master/src/ncp/ncp_base_mtd.cpp#L1011).

Reply:
Thanks. Updated.